cpp-ch/local-engine/Common/CHUtil.cpp

```diff
@@ -315,6 +316,16 @@ DB::Block BlockUtil::concatenateBlocksMemoryEfficiently(std::vector<DB::Block> &
     return out;
 }
 
+String BlockUtil::dumpBlock(const DB::Block & block)
```

Comment:
How are about moving to `BlockTypeUtils.h/.cpp` ?

Reply:
`BlockTypeUtils.h/.cpp` is just about types?

Reply:
Actually, it's block and type utils :)

Reply:
If dumping is just for debug aid, I think we should move it to `DebugUtils.h/cpp`, and put it to `debug` namespace. In product code, we should put it into` #ifdef NDEBUG ..  #endif`. e.g.

``` C++
#ifdef NDEBUG
 debug::dumpxxx();
#endif
```